Tests/InAppMessagingModuleSpec.swift

```diff
@@ -92,6 +95,84 @@ class InAppMessagingModuleSpec: QuickSpec {
                     }
                 }
 
+                context("and module is partially enabled") {
```

Comment:
I'm thinking of putting 2 loops, one with single 'will call...' and the other with 'will not call...'
The loops will iterate over (rollout, randomValue) combinations. What do you think @malayli?
It would look like this:
```
[(1, 1), (50, 49), (50, 50) ...].forEach { (rolloutPercentage, returnedValue) in
                    it("will call refreshList in CampaignsListManager") {
                         configurationManager.rolloutPercentage = rolloutPercentage
                         randomizer.returnedValue = returnedValue
                         iamModule.initialize { }

                         expect(campaignsListManager.wasRefreshListCalled).to(beTrue())
                    }
} 
```

Reply:
Is this to make the tests more concise?

Reply:
Yes

Reply:
🤓